editor/selectors.js

```diff
@@ -666,29 +666,28 @@ export function isTyping( state ) {
 }
 
 /**
- * Returns the unique ID of the block after which a new block insertion would
- * be placed, or null if the insertion point is not shown. Defaults to the
- * unique ID of the last block occurring in the post if not otherwise assigned.
+ * Returns the insertion point, the index at which the new inserted block would
+ * be placed. Defaults to the last position
  *
  * @param  {Object}  state Global application state
  * @return {?String}       Unique ID after which insertion will occur
  */
 export function getBlockInsertionPoint( state ) {
```

Comment:
Do we not have tests for this selector?

Reply:
Yes we do, and surprisingly they are passing because we're using `1,2,3` as uids which are the exact same indexes the function should return.